### PR TITLE
LLM : Add qlora cpu finetune docker image

### PR DESCRIPTION
**Pull request by hzjane**
## Description
 Add qlora cpu finetune docker image

### 4. How to test?
- [ ] N/A
- [ ] Unit test
- [ ] Application test
- [ ] Document test
- [ ] ...

### 5. New dependencies

- [ ] New Python dependencies
       - Dependency1 
       - Dependency2
       - ...
- [ ] New Java/Scala dependencies and their license
       - Dependency1 and license1
       - Dependency2 and license2
       - ...

### CODE REVIEW

**Comment by hzjane** [comment]
> Does CPU qolra work today?

Yes. Use wangyang's pr https://github.com/intel-analytics/BigDL/pull/9233/files, and remove ipex libs. The CPU qolra can work normally.

**Comment by jason-dai** [comment]
> > Does CPU qolra work today?
> 
> Yes. Use wangyang's pr https://github.com/intel-analytics/BigDL/pull/9233/files, and remove ipex libs. The CPU qolra can work normally.

Please submit a PR with examples for CPU QLoRA to `llm` first

File: docker/llm/finetune/qlora/cpu/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
+FROM intel/oneapi-basekit:2023.2.1-devel-ubuntu22.04
+ARG http_proxy
+ARG https_proxy
+ENV TZ=Asia/Shanghai
+ARG PIP_NO_CACHE_DIR=false
+ENV TRANSFORMERS_COMMIT_ID=95fe0f5
+
+# retrive oneapi repo public key
+RUN curl -fsSL https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2023.PUB | gpg --dearmor | tee /usr/share/keyrings/intel-oneapi-archive-keyring.gpg && \
```
Comment (glorysdj) [inline docker/llm/finetune/qlora/cpu/docker/Dockerfile:9]: Singe RUN to reduce image size

File: docker/llm/finetune/qlora/cpu/docker/README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+Enter the running container:
+
+```bash
+docker exec -it bigdl-llm-fintune-qlora-cpu bash
+```
+
+Then, start QLoRA fine-tuning:
+
+```bash
+bash start-qlora-finetuning-on-cpu.sh
```
Comment (glorysdj) [inline docker/llm/finetune/qlora/cpu/docker/README.md:73]: run bigdl-llm-init first?

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**